StdNonModSeq.md

```diff
@@ -128,10 +128,9 @@ Returns an iterator to the *first* element in `[first, last)` for which the pred
   };
   ```
 
-
 ### Return
 
-- (1,2): `InputIterator` instance pointing to the first element satisfying condition, or `last` if no elements is found
+- (1,2): `InputIterator` instance pointing to the first element where the predicate is evaluated to true, or `last` if no element is found
```

Comment:
What was wrong with the original version?  I would have just done "elements -> such element"